Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -873,10 +877,13 @@ public sealed class TransformSpeakerNameEvent : EntityEventArgs
     public EntityUid Sender;
     public string Name;
 
-    public TransformSpeakerNameEvent(EntityUid sender, string name)
+    public SpeechVerbPrototype? SpeechVerb;
```

Comment:
like i mentioned before, you ideally shouldn't be passing around prototypes. Just get the ID and then index it when necessary. This also saves event subscribers from having to get PrototypeManager to index it.